src/theory/quantifiers/term_database_sygus.h

```diff
@@ -83,6 +83,9 @@ class TermDbSygus {
 private:
  /** mapping from enumerator terms to the conjecture they are associated with */
  std::map<Node, CegConjecture*> d_enum_to_conjecture;
+ /** mapping from enumerator terms to the function-to-synthesize they are
+  * associated with */
+ std::map<Node, Node> d_enum_to_synth_fun;
```

Comment:
Is the order of the elements needed here? If not used unordered_map here.

Reply:
The order is potentially helpful here, since it means I can fix an ordering in which I process enumerators.